### PR TITLE
feat(be): bug fix about catalog

### PR DESCRIPTION
**Pull request by o0OooO**
**What is the current behavior?

Please describe the current behavior of the code that you are trying to modify. This could include any issues or limitations that you are addressing.

**What is the updated behavior?

Please describe the changes you are making to the code's behavior. This could include new features or enhancements that you are adding, as well as how they differ from the current behavior.

**Checklist

Please check off the following items before submitting your pull request:

- [x] I have tested my changes.
- [ ] No sensitive information is included.
- [ ] I've reviewed my changes to ensure they won't cause any new issues that could affect the stability or performance of the codebase.

Please check off the following items if this changes include API modefications:

- [ ] I confirm this changes has been test with [authorization validation steps](https://github.com/awslabs/sensitive-data-protection-on-aws/blob/main/CONTRIBUTING.md#contributing-via-pull-requests), it won't break the authorization token verification mechanism.
- [ ] I confirm this changes won't cause security issues.

### CODE REVIEW

**Review by yanbasic — APPROVED** [review]
LGTM

**Comment by NingLu** [comment]
We identified new issues on unchanged lines of code. Navigate to the Amazon CodeGuru Reviewer console to view the recommendations to fix them.

File: source/constructs/api/data_source/service.py [hunk 1]
```diff
@@ -2026,38 +2026,32 @@ def list_data_location():
...
+            # location = DataLocationInfo()
+            # location.source = item.provider_name
+            # location.region = None
+            # location.coordinate = None
+            # location.account_count = 0
+            # res.append(location)
             continue
         for subItem in regions:
+            accounts = crud.list_account_by_provider_and_region(item.id, subItem.region_name)
+            if len(accounts) == 0:
```
Comment (NingLu) [inline source/constructs/api/data_source/service.py:2038]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)

**Review by yanbasic — APPROVED** [review]
LGTM

File: source/constructs/api/catalog/crud.py [hunk 2]
```diff
@@ -935,21 +943,38 @@ def get_export_catalog_data():
...
+               models.CatalogColumnLevelClassification.column_name,
+               models.CatalogColumnLevelClassification.identifier,
+               models.CatalogColumnLevelClassification.column_value_example,
+               models.CatalogTableLevelClassification.label_ids,
+               models.CatalogDatabaseLevelClassification.label_ids,
+               models.CatalogColumnLevelClassification.comments).all()
+
+
+def get_catalog_summay_by_provider_region(region: str):
+    return (get_session().query("database_type"),
```
Comment (NingLu) [inline source/constructs/api/catalog/crud.py:968]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears you are unpacking more than three variables while using a function that returns more than one variable. Using a large number of return values is prone to errors. We recommend that you return a small class or `namedtuple` instance instead.

[Learn more](https://docs.python.org/3/library/typing.html#typing.NamedTuple)

**Review by yanbasic — APPROVED** [review]
LGTM

**Pull request merged**